src/Serializer/ArraySerializerWithNull.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace League\Fractal\Serializer;
+
+class ArraySerializerWithNull extends ArraySerializer
```

Comment:
I'm not sure we need a separate class for this. Can you walk me through your reasoning for not just adding `null()` to `ArraySerializer`?